frontend/awx/administration/instances/hooks/useInstancesColumns.tsx

```diff
@@ -94,6 +113,16 @@ export function useInstancesColumns(options?: { disableSort?: boolean; disableLi
           ),
         list: 'secondary',
       },
+      {
+        header: t('Running jobs'),
```

Comment:
It seems that Expanded option is missing in PageTable and I cant see Running and Total jobs, there must be another  switch to turn it on, possibly in PageTable?

Reply:
Yeah I've been struggling to find out how to add the column to the expanded section I thought `ColumnTableOption.expanded` would work. Might be a framework bug?

Reply:
Was thinking in the interest of time just to fix this during hardening or later in another PR. I can go ahead and make a Jira issue for it